mint.json

```diff
@@ -1,262 +1,268 @@
-            "api-reference/public/webauthn/deletes-a-webauthn-credential",
-            "api-reference/public/webauthn/updates-a-webauthn-credential"
-          ]
-        },
-        {
-          "group": ".well-known",
-          "pages": [
-            "api-reference/public/well-known/get-json-web-key-set",
-            "api-reference/public/well-known/get-public-hanko-configuration"
-          ]
-        },
-        {
-          "group": "Third Party",
-          "pages": [
-            "api-reference/public/third-party/initialize-third-party-login",
-            "api-reference/public/third-party/third-party-provider-callback"
-          ]
-        },
-        {
-          "group": "Token",
-          "pages": [
-            "api-reference/public/token/exchange-one-time-token-for-session"
-          ]
-        },
-        {
-          "group": "User Management",
-          "pages": [
-            "api-reference/public/user-management/get-user-details-by-email",
-            "api-reference/public/user-management/deletes-the-current-user",
-            "api-reference/public/user-management/get-the-current-user-id",
-            "api-reference/public/user-management/log-out-the-current-user",
-            "api-reference/public/user-management/create-a-user",
-            "api-reference/public/user-management/get-a-user-by-id"
-          ]
-        },
-        {
-          "group": "Email Management",
-          "pages": [
-            "api-reference/public/email-management/get-a-list-of-emails-of-the-current-user",
-            "api-reference/public/email-management/add-a-new-email-address-to-the-current-user",
-            "api-reference/public/email-management/marks-the-email-address-as-primary-email",
-            "api-reference/public/email-management/delete-an-email-address"
-          ]
-        }
-      ]
-    },
-    {
-      "group": "Admin",
-      "pages": [
-        {
-          "group": "Status",
-          "pages": ["api-reference/admin/status/status-page"]
-        },
-        {
-          "group": "User Management",
-          "pages": [
-            "api-reference/admin/user-management/get-a-list-of-users",
-            "api-reference/admin/user-management/create-a-new-user",
-            "api-reference/admin/user-management/get-a-user-by-id",
-            "api-reference/admin/user-management/delete-a-user-by-id"
-          ]
-        },
-        {
-          "group": "Audit Logs",
-          "pages": ["api-reference/admin/audit-logs/get-a-list-of-audit-logs"]
-        },
-        {
-          "group": "Metrics",
-          "pages": ["api-reference/admin/metrics/get-prometheus-metrics"]
-        }
-      ]
-    }
-  ],
-  "footerSocials": {
-    "twitter": "https://twitter.com/hanko_io",
-    "github": "https://github.com/teamhanko",
-    "linkedin": "https://www.linkedin.com/company/teamhanko/"
-  },
-  "analytics": {
-    "plausible": {
-      "domain": "docs.hanko.io"
-    }
-  }
+	"name": "Hanko",
```

Comment:
The formatting for the whole mint.json, seems to be a bit messed up. Can you please make sure you're using prettier to format

Reply:
I am using Prettier -- that's weird. let me check whats going on

Reply:
Fixed. I've added a .prettierrc.json at the top level so it doesn't take my VSCode's prettier config, but just the standard one (I have `useTabs` set to `true` by default)